src/api/accounts.rs

```diff
@@ -82,21 +87,37 @@ mod accounts_signing {
                 };
             }
             let from = key.address();
+
+            let gas_price = match tx.transaction_type {
+                Some(tx_type) if tx_type == U64::from(2) && tx.max_fee_per_gas.is_some() => tx.max_fee_per_gas,
```

Comment:
Can we move the `U64::from(2)` or just `2` into a constant?

Reply:
Constant.